src/frontend/screens/TrackRecordingActive.tsx

```diff
@@ -0,0 +1,75 @@
+import {View} from 'react-native';
+import {BottomSheetWrapper} from '../sharedComponents/BottomSheetWrapper';
+import {HeaderText} from '../sharedComponents/Text/HeaderText';
+import {defineMessages, useIntl} from 'react-intl';
+import {BodyText} from '../sharedComponents/Text/BodyText';
+import {DestructiveButton, SecondaryButton} from '../sharedComponents/Buttons';
+import {NativeRootNavigationProps} from '../sharedTypes/navigation';
+import Error from '../images/Error.svg';
+import {useTracking} from '../hooks/useTracking';
+
+const m = defineMessages({
+  recordingTracks: {
+    id: 'screens.TrackRecordingActive.recordingTracks',
+    defaultMessage: 'Recording Tracks!',
+  },
+  warningMessage: {
+    id: 'screens.TrackRecordingActive.warningMessage',
+    defaultMessage:
+      'You’re currently recording a track. To join project, stop recording',
+  },
+  stopTracks: {
+    id: 'screens.TrackRecordingActive.stopTracks',
+    defaultMessage: 'Stop Tracks',
+  },
+  cancel: {
+    id: 'screens.TrackRecordingActive.cancel',
+    defaultMessage: 'Cancel',
+  },
+});
+
+export const TrackRecordingActive = ({
+  navigation,
+}: NativeRootNavigationProps<'TrackRecordingActive'>) => {
+  const {formatMessage} = useIntl();
+  const {cancelTrackingAndReturnIfTracksSaved} = useTracking();
+
+  function handleStopTracks() {
+    const hasTracksSaved = cancelTrackingAndReturnIfTracksSaved();
+
+    if (hasTracksSaved) {
+      navigation.replace('SaveTrack');
```

Comment:
maybe more of a UX question, but even if the track is saved, the user loses access to it if they eventually accept the invite, right? giving the option of saving something that they won't be able to access seems like it could lead to confusion.

Reply:
Decided it was more suitable to ask this in the original issue: https://github.com/digidem/comapeo-mobile/issues/1089#issuecomment-2852259110